gridy/Sources/Presentations/PlanBoard/BoardSettingView.swift

```diff
@@ -8,26 +8,156 @@
 import SwiftUI
 
 struct BoardSettingView: View {
-    @Environment(\.dismiss) private var dismiss
-
+    @State var projectName = ""
+    @State var cancelHover = false
+    @State var createHover = false
+    @State var startDateHovered = false
+    @State var endDateHovered = false
+    @State var selectedStartDate = Date()
+    @State var selectedEndDate = Date()
+    @State var startDatePickerPresented = false
+    @State var endDatePickerPresented = false
+    
     var body: some View {
-        NavigationStack {
-            Text("BoardSettingView")
-                .scenePadding()
-        }
-        .toolbar {
-            ToolbarItem(placement: .cancellationAction) {
-                Button("Cancel") {
-                    dismiss()
+        VStack(alignment: .center, spacing: 24) {
+            folderNameTextField
+            HStack(alignment: .center, spacing: 16) {
+                startDuration
+                endDuration
+            }
+            HStack(alignment: .center, spacing: 8) {
+                cancel
+                create
+            }
+        }
+        .padding(24)
+        .frame(width: 480)
+    }
+}
+
+extension BoardSettingView {
+    var folderNameTextField: some View {
+        RoundedRectangle(cornerRadius: 16)
+            .foregroundStyle(Color.item)
+            .frame(height: 48)
+            .overlay(
+                TextField("Project Name", text: $projectName)
+                    .textFieldStyle(.plain)
+                    .padding(.horizontal, 16)
+            )
+    }
+}
+
+extension BoardSettingView {
+    var startDuration: some View {
+        Button {
+            startDatePickerPresented = true
+        } label: {
+            Text("시작 : \(formattedDate(date: selectedStartDate))")
+                .foregroundStyle(Color.title)
+                .frame(width: 132, height: 32)
+                .background(
+                    RoundedRectangle(cornerRadius: 5)
+                        .foregroundStyle(startDateHovered ? Color.itemHovered : Color.clear)
+                )
+                .onHover { hover in
+                    startDateHovered = hover
                 }
+        }
+        .buttonStyle(.link)
+        .popover(isPresented: $startDatePickerPresented) {
+            VStack {
+                DatePicker("", selection: $selectedStartDate, displayedComponents: [.date])
+                    .datePickerStyle(GraphicalDatePickerStyle())
+                    .labelsHidden()
+                Button("Done", action: {
+                    startDatePickerPresented = false
+                })
             }
```

Comment:
지난 코드리뷰에서 후행클로저에 대한 코멘트를 남긴 적이 있습니다
```suggestion
                Button("Done") {
                    startDatePickerPresented = false
                }
```

Reply:
[fix][#62] Button 수정

<img width="134" alt="스크린샷 2023-11-07 오후 3 49 53" src="https://github.com/DeveloperAcademy-POSTECH/MacC-Team15-Ampersand/assets/125735850/45965e9a-a659-4303-94d8-7c9b2c3ef9f1">